Dockerfile

```diff
@@ -1,25 +1,36 @@
 #
 # Python Dependencies
 #
-FROM alpine:3.10.2 as python
+FROM alpine:3.11.3 as python
 
 RUN sed -i 's|http://dl-cdn.alpinelinux.org|https://alpine.global.ssl.fastly.net|g' /etc/apk/repositories
-RUN apk add python python-dev py-pip py-virtualenv libffi-dev gcc linux-headers musl-dev openssl-dev make
+RUN apk add python3 python3-dev libffi-dev gcc linux-headers musl-dev openssl-dev make
 
 COPY requirements.txt /requirements.txt
 
-RUN pip install -r /requirements.txt --install-option="--prefix=/dist" --no-build-isolation
+RUN python3 -m pip install --upgrade pip setuptools wheel && \
+    pip install -r /requirements.txt --ignore-installed --prefix=/dist --no-build-isolation --no-warn-script-location
+
+# Crudini needs special handling because it depends on the only marginally maintained iniparse
+RUN cd /tmp && wget https://github.com/candlepin/python-iniparse/archive/727a5d212d79bddcd6fb1116e406700be63ded24.zip && \
```

Comment:
Let's simplify. All the wget and unzipping unnecessary with pip. It can install URLs. Feel free to move `727a5d212d79bddcd6fb1116e406700be63ded24` to an ENV.

```suggestion
RUN pip install https://github.com/candlepin/python-iniparse/archive/727a5d212d79bddcd6fb1116e406700be63ded24.zip --ignore-installed --prefix=/dist --no-build-isolation
```

![image](https://user-images.githubusercontent.com/52489/74207428-7bf6be80-4c34-11ea-83fa-dbf162791e3c.png)

Reply:
I did not know that `pip` could install directly from an URL.